cmd/duffle/claims_show.go

```diff
@@ -15,42 +17,66 @@ Display the content of a claim.
 This dumps the entire content of a claim as a JSON object.
 `
 
+type claimsShowCmd struct {
+	Name       string
+	OnlyBundle bool
+	Output     string
+	Storage    claim.Store
+}
+
 func newClaimsShowCmd(w io.Writer) *cobra.Command {
-	var onlyBundle bool
+	var cmdData claimsShowCmd
 	cmd := &cobra.Command{
 		Use:     "show NAME",
 		Short:   "show a claim",
 		Long:    claimsShowDesc,
 		Aliases: []string{"get"},
 		Args:    cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			name := args[0]
-			storage := claimStorage()
-			return displayClaim(name, w, storage, onlyBundle)
+			cmdData.Name = args[0]
+			cmdData.Storage = claimStorage()
+			return cmdData.runClaimShow(w)
 		},
 	}
 
-	cmd.Flags().BoolVarP(&onlyBundle, "bundle", "b", false, "only show the bundle from the claim")
+	cmd.Flags().BoolVarP(&cmdData.OnlyBundle, "bundle", "b", false, "only show the bundle from the claim")
+	cmd.Flags().StringVarP(&cmdData.Output, "output", "o", "", "show the contents of the named output")
 
 	return cmd
 }
 
-func displayClaim(name string, out io.Writer, storage claim.Store, onlyBundle bool) error {
-	c, err := storage.Read(name)
+func (csc claimsShowCmd) runClaimShow(w io.Writer) error {
+	if csc.OnlyBundle && csc.Output != "" {
+		return errors.New("invalid flags: at most one of [--bundle, --output] can be specified")
```

Comment:
Is the following more readable?
```suggestion
		return errors.New("invalid flags: at most one of --bundle and --output can be specified")
```